docs/api/ReduxForm.md

```diff
@@ -31,7 +31,7 @@ or passed in as props to your component at runtime.**
 #### -`asyncBlurFields : Array<String>` [optional]
 
 > field names for which `onBlur` should trigger a call to the `asyncValidate` function.
-Defaults to `[]`.
+Defaults to including all fields.
```

Comment:
It's not actually. An empty array will result in no fields triggering the async blur: https://github.com/erikras/redux-form/blob/master/src/createReduxForm.js#L448